package.json

```
@@ -67,7 +67,7 @@
   },
   "scripts": {
     "show-linked-packages": "find node_modules node_modules/@* -depth 1 -type l -print",
-    "link-packages": "grep -oE '@shapeshiftoss\/[a-z-]*' package.json | grep -v hdwallet | xargs yarn link",
+    "link-packages": "grep -oE '@shapeshiftoss/[a-z-]*' package.json | grep -v hdwallet | xargs yarn link",
```

yarn keeps removing this backslash. I even tried not to commit this change but after running yarn a few times it snuck back in.

```suggestion
    "link-packages": "grep -oE '@shapeshiftoss\/[a-z-]*' package.json | grep -v hdwallet | xargs yarn link",
```